notebooks/239-freevc/239-freevc.ipynb

```diff
@@ -0,0 +1,761 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #65.        :param return_partials: if True, the partial embeddings will also be returned along with
</pre><p>smodel missed in docstring, could you please add?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1042/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>